TodoList/Src/TodoList.Dependency/Containers/TodoListContainer.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using TodoList.Contracts.DI;
+using TodoList.DI.Extensions;
+using Unity;
+using Unity.Exceptions;
+using Unity.Injection;
+
+namespace TodoList.DI.Containers
```

Comment:
Still abbreviations in namespaces? 😜 
```suggestion
namespace TodoList.DependencyInjection.Containers
```
```suggestion
namespace TodoList.Dependency.Containers
```
```suggestion
namespace TodoList.Injection.Containers
```
```suggestion
namespace TodoList.InversionOfControl.Containers
```

Reply:
You should create a subfolder for each dot-separated part of a namespace → I miss `Containers` folder 😉 
On the other hand, I believe this last part of the namespace is a bit redundant and leaving it out completely won't hurt anything…
```suggestion
namespace TodoList.DependencyInjection
```
… (or any other variant above) …
```suggestion
namespace TodoList.Container
```
But I would still stick with `Dependency`, based on your implementation of this contract 😄